src/main/resources/templates/db_scripts/schema.json

```diff
@@ -289,6 +289,48 @@
           "targetTableAlias": "fiscalYear"
         }
       ]
+    },
+    {
+      "tableName": "groupFundFY",
+      "fromModuleVersion": 4.0,
+      "foreignKeys": [
+        {
+          "fieldName": "groupId",
```

Comment:
Since we are adding a foreign key, shouldn't we add `"tOps": "ADD"` here? This applies to all foreignKeys below.